arm-sql/2014-04-01/examples/DataMaskingPolicyCreateOrUpdateMin.json

```diff
@@ -0,0 +1,47 @@
+{
+   "parameters": {
+      "subscriptionId": "00000000-1111-2222-3333-444444444444",
+      "resourceGroupName": "sqlcrudtest-6852",
+      "serverName": "sqlcrudtest-2080",
+      "databaseName": "sqlcrudtest-331",
+      "dataMaskingPolicyName": "Default",
+      "api-version": "2014-04-01",
+      "parameters": {
+         "properties": {
+            "dataMaskingState": "Enabled"
+         }
+      }
+   },
+   "responses": {
+      "201": {
```

Comment:
Yeah, I had 201 in case it was likely to change, but looks like it's pretty solidified so I'll remove